index.md

```diff
@@ -144,7 +144,7 @@ Riot.js has between 10 and 100 times fewer API methods than other UI libraries.
 
 {% include libraries_comparison.html %}
 
-- Only 6kb!
+- Only 7kb!
```

Comment:
Based on what on what's displayed in the chart and on [bundlephobia](https://bundlephobia.com/result?p=riot@4.12.1)